components/button/README.md

```diff
@@ -19,6 +19,16 @@ The `d2l-button` element can be used just like the native button element, but al
 - `disabled` (Boolean): disables the button
 - `primary` (Boolean): styles the button as a primary button
 
+**Accessibility:**
+
+To make your `d2l-button` accessible to screen reader users, use the following properties when applicable:
```

Comment:
Accessibility applies to more than just screen readers, so I'd drop that bit and just say:

> To make your `d2l-button` accessible, use the following properties when applicable: